Rakefile

```diff
@@ -7,27 +7,31 @@ Rake::TestTask.new(:integration_test) do |t|
   t.libs << "test"
   t.libs << "lib"
   t.test_files = FileList['test/integration/**/*_test.rb']
+  t.warning = false
```

Comment:
Don't want to miss anything important here, but maybe we can silence the gems that are causing these warnings that are outside of our control. Either way I don't think this should be part of this PR.